core/src/main/java/org/bitcoinj/core/PeerGroup.java

```diff
@@ -2135,16 +2135,7 @@ public void triggerPreBlockDownloadComplete() {
     private Integer masternodeListsSynced = 0;
     public void triggerMnListDownloadComplete() {
         masternodeListsSynced++;
-        int requiredListsToSync = LLMQUtils.isQuorumRotationEnabled(chain, params, params.getLlmqDIP0024InstantSend()) ? 2 : 1;
-
-        log.info("triggerMnListDownloadComplete: {}, requiredListsToSync = {}", masternodeListsSynced + 1, requiredListsToSync);
-        if (LLMQUtils.isQuorumRotationEnabled(context, params, params.getLlmqDIP0024InstantSend())) {
-            if (masternodeListsSynced == requiredListsToSync) {
-                mnListDownloadedFuture.set(masternodeListsSynced);
-            }
-        } else {
-            mnListDownloadedFuture.set(masternodeListsSynced);
-        }
+        mnListDownloadedFuture.set(masternodeListsSynced);
```

Comment:
Originally this part would be triggered for both "mnlistdiff" and "qrinfo" messages and both were required to signal the end of the masternode list sync process by setting `mnListDownloadedFuture`.

Now we only want to include `mnlistdiff` in the masternode list sync.